contracts/payment/PullPayment.sol

```diff
@@ -1,43 +1,43 @@
-pragma solidity ^0.4.21;
-
-
-import "../math/SafeMath.sol";
+pragma solidity ^0.4.24;
 
+import "./Escrow.sol";
+import "zos-lib/contracts/migrations/Migratable.sol";
 
 /**
  * @title PullPayment
  * @dev Base contract supporting async send for pull payments. Inherit from this
- * contract and use asyncSend instead of send or transfer.
+ * contract and use asyncTransfer instead of send or transfer.
  */
-contract PullPayment {
-  using SafeMath for uint256;
+contract PullPayment is Migratable {
+  Escrow private escrow;
 
-  mapping(address => uint256) public payments;
-  uint256 public totalPayments;
+  function initialize() public isInitializer("PullPayment", "1.11.0") {
+    escrow = new Escrow();
```

Comment:
Note: this probably makes deploying `zos`'s `PullPayment` _much_ more expensive than OZ's, because it now needs to be able to create an `Escrow` in any TX, as opposed to only during construction, and therefore its bytecode will now need to be stored.